src/renderer/mixins/DateFormatter.js

```diff
@@ -0,0 +1,43 @@
+const DateFormatter = {
+  methods: {
+    toLocalePublicationString (videoData) {
```

Comment:
Named object parameters would probably be better here
Ex:
```suggestion
    toLocalePublicationString ({ publishedText, publishedDate, isLive = false, isUpcoming = false, isRSS = false }) {
```

Reply:
Yep, you're right. I'm not sure how to commit this suggestion, because it would mean that the references to `videoData` would have to be changed as well.

Reply:
You'd have to make the change in your code editor